SpotSwap/Services/FireBase Services/DataBase Service/DataBaseService+EndUsers.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  DataBaseService+EndUsers.swift
+//  SpotSwap
+//
+//  Created by Yaseen Al Dallash on 3/14/18.
+//  Copyright © 2018 Yaseen Al Dallash. All rights reserved.
+//
+
+import Foundation
+import Firebase
+extension DataBaseService{
+    //This will add a new EndUser to the dataBase using the user UID
+    func addNewEndUser(endUser: EndUser, user: User, completion: ()->Void, errorHandler: (Error)->Void){
```

Comment:
What is the difference between EndUser and User? Caroline would like you to be more descriptive.